src/autopas/utils/StringUtils.h

```diff
@@ -251,6 +251,55 @@ inline std::set<int> parseInts(const std::string &intString) {
   return ints;
 }
 
+template <class N>
```

Comment:
```suggestion
template <class T>
```
Nothing wrong with `N`, but `T` is conventional.